electron/app/services/cluster/ipc.ts

```diff
@@ -0,0 +1,8 @@
+import {PingParams} from '@shared/ipc';
+
+import {handleIpc} from '../../utils/ipc';
+import {ping} from './handlers/ping';
+
+handleIpc('cluster:ping', (params: PingParams) => {
```

Comment:
Step 2. You define your handler helper like this.

Errors thrown from the handler will be properly forwarded to the renderer and rethrown on that side.